packages/webapp/src/delegates/components/delegate-funds-available.tsx

```diff
@@ -84,6 +91,42 @@ export const DelegateFundsAvailable = ({ account }: Props) => {
                     )}
                 </div>
             </div>
+            {isLoggedMember && <NextDisbursementInfo />}
         </>
     );
 };
+
+export const NextDisbursementInfo = () => {
+    const { data: distributionState } = useDistributionState();
+
+    if (!distributionState) return null;
+
+    switch (distributionState.state) {
+        case "next_distribution":
+            return (
+                <Text>
+                    Delegate funds are disbursed monthly. Check back on{" "}
+                    {dayjs(
+                        distributionState.data.distribution_time + "Z"
+                    ).format("LLL")}{" "}
+                    for your next disbursement.
+                </Text>
+            );
+        case "election_distribution":
+            return (
+                <Text>
+                    An election is running and a disbursement will be made as
+                    soon as the election ends.
+                </Text>
```

Comment:
I recommend we tweak the language here to reflect that they only get this next disbursement if they come out of the ongoing election as a delegate:

> An election is currently underway. Disbursements to newly-elected delegates will be processed as soon as the election ends.